dynamic-plugins/tekton-plugins.yaml

```diff
@@ -0,0 +1,45 @@
+plugins:
```

Comment:
Question: the remaining dynamic plugins will probably be installed with the rhdh config script? such as topology

Reply:
@Jdubrick Yes, I'll create a `developer-hub-plugins.yaml` or `plugins.yaml` in the changes for the next issue.